enterprise/frontend/src/embedding-sdk/components/public/InteractiveDashboard/EditableDashboard.stories.tsx

```diff
@@ -7,7 +7,13 @@ import {
   type EditableDashboardProps,
 } from "./EditableDashboard";
 
-const DASHBOARD_ID = (window as any).DASHBOARD_ID || 1;
+const ENTITY_ID = "xBLdW9FsgRuB2HGhWiBa_";
```

Comment:
these are the arguments I used when testing and fixing this bug. Maybe we won't need it anymore, thought? The same pattern is applied to all other component test files.